example/e2e/functional.test.ts

```diff
@@ -180,6 +192,10 @@ describe('E2E Functional Tests', () => {
   });
 
   it('should return error message upon a failed 3DS2 payment transaction', async () => {
+    await launchApp(defaultConfig);
+    if (await isAndroid()) {
+      await device.disableSynchronization();
+    }
```

Comment:
You have these lines repeated in many places, maybe extracting them as a separate function to save some space and reduce cognitive complexity will be a nice thing to do